proto/viam/app/v1/app.proto

```diff
@@ -114,7 +132,10 @@ message RobotPartHistoryEntry {
   RobotPart old = 4 [(tagger.v1.tags) = "bson:\"old\" json:\"old\""];
 }
 
-message ListOrganizationsRequest {}
+message ListOrganizationsRequest {
+  // Returns only organizations that are shared with a specific Location ID.
+  optional string shared_with_location_id = 3;
```

Comment:
This feels funky personally. Did you discuss this elsewhere?

Reply:
Yeah it was a tradeoff. I will send you the slack thread about it.

Reply:
ok. Updated to remove this field as we will handle in our frontend.